api/CSSMediaRule.json

```diff
@@ -54,40 +54,40 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMediaRule/media",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "1"
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤3"
```

Comment:
```
              "version_added": "≤3"
```

Wasn’t the decision to do with `≤4` rather than `≤3`? (I can’t remember of that was the resolution or not…)

Reply:
In that PR, I added both ranged versions.  Since the VMs I have can go as far back as Safari 3 in an efficient manner, I figured it would make sense to keep them both for more real data sooner.

Reply:
> In that PR, I added both ranged versions. Since the VMs I have can go as far back as Safari 3 in an efficient manner, I figured it would make sense to keep them both for more real data sooner.

Got it — thanks for clarifying

Reply:
I was also able to get a new set of Safari 1-2 emulators working that function much, much better than the last ones did, which means more testing opportunities, and clearing the ranged versions in this PR, so this is now ready to go!